CHANGELOG.md

```diff
@@ -2,335 +2,355 @@
+Implementers need to configure these changes it in your scheduler task system in the production server. We give the examples with `crontab`, although alternatively you could use `whenever` gem or the scheduled jobs of your hosting provider.
 
-### Changed
+#### 4.1. Reminders for participants
 
-#### Accept and reject cookies
+We have added the possibility to send reminders for some actions, like pending budgets orders or user generated meetings that weren't closed.
 
-Cookie consent management has been updated in [\#9271](https://github.com/decidim/decidim/pull/9271). Supported cookie categories are essential, preferences, analytics and marketing.
-Iframe HTML elements that are added with the editor or meeting forms are disabled until all cookies are accepted. Scripts that require cookies could be added as follows:
-
-```html
-<script type="text/plain" data-consent="marketing">
-  console.log('marketing cookies accepted');
-</script>
+```console
+# Generate reminders
+4 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:reminders:all
 ```
 
-Note that you need to define the `type="text/plain"` for the script that adds cookies in order to prevent the script from being executed before cookies are accepted. You should also define the metadata for all the cookies that you're using on your app initializer. See [cookie documentation](https://docs.decidim.org/en/customize/cookies.html).
+You can read more about this change on PR [\#8621](https://github.com/decidim/decidim/pull/8621).
 
+#### 4.2. Mail Notifications digest
 
-Mind that we also changed the cookie consent cookie from "decidim-cc" to "decidim-consent" by default. You can change it on your initializer, or update your cookie legal notice accordingly.
+Participants can configure if they want to receive notifications in real-time (one email by any action that they're notified to), or a daily or weekly notifications digest (a highlight with some of the notifications).
 
-#### Rename data portability to download your data
+```console
+# Send notification mail digest daily
+5 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:mailers:notifications_digest_daily
 
-"Data portability" has been renamed to "Download you data" at [\#9196](https://github.com/decidim/decidim/pull/9196), you should update your cron job via crontab -e.
+# Send notification mail digest weekly on saturdays
+5 0 * * 6 cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:mailers:notifications_digest_weekly
 ```
+
+You can read more about this change on PR [\#8833](https://github.com/decidim/decidim/pull/8833).
+
+#### 4.3. Rename data portability to download your data
+
+"Data portability" has been renamed to "Download you data". As this was a scheduled task that was already configured you'll need to change it. Where you had:
+
+```console
+# Remove expired download your data files
```

Comment:
One more thing I noticed. This may be easier to locate if we use the legacy comment in the section that refers to the old cronjob:
```suggestion
# Remove expired data portability files
```

After this, it's all good for me.

Reply:
Makes sense!